### PR TITLE
**Feature:**  add a way to customise colours for tabs

### PR DESCRIPTION
**Pull request by stereobooster**
# Summary

Screenshot 2019-08-02 at 17 30 05

# Related issue

N/A

# To be tested

Me
- [ ] No error or warning in the console on `localhost:6060`

Tester 1

- [ ] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

File: src/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -154,6 +155,7 @@ const Tabs: React.FC<TabsProps> = ({
                 key={i}
                 onClick={onClick}
                 ref={i === active ? activeTab : undefined}
+                color={color}
```
Comment (fabien0102) [inline src/Tabs/Tabs.tsx]: You should use `expandColor` from constant.ts to have all the theme colors ;)

File: src/Tabs/README.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ const MyComponent = () => {
   const [tabs, setTabs] = React.useState([
     {
       title: "tab 1 tab 1 tab 1 tab 1 tab 1 tab 1 tab 1 tab 1",
+      color: "#e2eff5",
```
Comment (TejasQ) [inline src/Tabs/README.md]: Would `color: "primary"` work here? (It should)

File: src/Tabs/Tabs.styled.ts [hunk 3]
```diff
@@ -46,17 +49,19 @@ export const TabHeader = styled(SectionHeader, {
   condensed?: boolean
   as?: React.FC<any> | string
   center?: boolean
+  color?: string
 }>`
   justify-content: ${({ center }) => (center ? "center" : "space-between")};
   cursor: pointer;
   font-weight: normal;
-  background-color: ${({ theme }) => theme.color.background.light};
+  background-color: ${({ theme, color }) => (color ? darken(color, 10) : theme.color.background.light)};
```
Comment (TejasQ) [inline src/Tabs/Tabs.styled.ts]: Use `lighten` and `expandColor`.

File: src/Tabs/README.md (hunk 2 above)
Comment (stereobooster) [reply src/Tabs/README.md]: Primary would be too dark. This is the different colour. As well this is just a readme, so I can choose any colour for the example

**Comment by stereobooster** [comment]
I added expandColor

**Comment by stereobooster** [comment]
Thanks @fabien0102 and @TejasQ

**Pull request merged**